tests/integration/run_tests.py

```diff
@@ -821,6 +840,132 @@ async def test_getting_all_objects_from_schema_works(sess, folder_id):
     await asyncio.gather(*[delete_object(sess, "study", accession_id) for accession_id, _ in files])
 
 
+async def test_metax_crud(sess, folder_id):
+    """Test Metax service with study and dataset POST, PATCH, PUBLISH and DELETE reqs.
+
+    :param sess: HTTP session in which request call is made
+    :param folder_id: id of the folder where objects reside
+    """
+    # POST to object endpoint creates draft dataset in Metax for Study and Dataset
+    ids = []
+    xml_files = set()
+    for schema, filename, update_filename in {
+        ("study", "SRP000539.xml", "SRP000539_put.xml"),
+        ("dataset", "dataset.xml", "dataset_put.xml"),
+    }:
+        accession_id, _ = await post_object(sess, schema, folder_id, filename)
+        xml_files.add((schema, accession_id, update_filename))
+        ids.append([schema, accession_id])
+
+    json_files = set()
+    for schema, filename, update_filename in {
+        ("study", "SRP000539.json", "patch.json"),
+        ("dataset", "dataset.json", "dataset_patch.json"),
+    }:
+        accession_id = await post_object_json(sess, schema, folder_id, filename)
```

Comment:
if we use the `patch.json` and `dataset_patch.json` `patch_object_json` would be more appropriate ?

if we keep patch this will fail in #332 as we will only have one study per forlder

Reply:
I try to fix those in #332 if you don't want to fix these as part of this issue.